guides/common/modules/proc_integrating-powerdns.adoc

```diff
@@ -0,0 +1,20 @@
+[id="integrating-powerdns"]
+= Integrating PowerDNS
+
+The _dns_powerdns_ DNS provider manages DNS records using the https://www.powerdns.com/[PowerDNS] REST API.
+
+.Procedure
+. Configure {ProjectServer} or {SmartProxyServer} to connect to the PowerDNS service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-dns true \
+--foreman-proxy-dns-provider powerdns \
+--enable-foreman-proxy-plugin-dns-powerdns \
+--foreman-proxy-plugin-dns-powerdns-rest-api-key _api_key_ \
+--foreman-proxy-plugin-dns-powerdns-rest-url http://_powerdns_hostname_or_ip_address_:8081/api/v1/servers/localhost
```

Comment:
```suggestion
--foreman-proxy-plugin-dns-powerdns-rest-url http://_powerdns.example.com_:8081/api/v1/servers/localhost
```

question: Would this also work with HTTPS?

Reply:
It would, but AFAIK PowerDNS itself doesn't offer HTTPS. You need to set up a reverse proxy.

Reply:
I had the same idea when I edited this file and read "http" but @ekohl is right. [You need a reverse proxy](https://dnssec-script.com/configuring-ssl-for-powerdns-api/). Since they don't support HTTPS without external software, we should keep HTTP. I wouldn't work if users copy/paste the command and only change the hostname.